packages/core/test/unit/segment_emitter.test.js

```diff
@@ -87,8 +87,8 @@ describe('SegmentEmitter', function() {
         SegmentEmitter.disableReusableSocket();
       }
 
-      it('should send the segment using the dgram client', testSegmentSend.bind(undefined, 1, 2, configureHook));
-      it('should share the dgram client between many segments sent at once', testSegmentSend.bind(undefined, 10, 3, configureHook));
+      it('should send the segment using the dgram client', testSegmentSend.bind(undefined, 1, 1, configureHook));
+      it('should share the dgram client between many segments sent at once', testSegmentSend.bind(undefined, 10, 2, configureHook));
```

Comment:
Since `createSocket` is no longer called immediately on require (and then discarded when `disableReusableSocket()` is called), we can decrement the number of expected calls to `createSocket` by one for these tests.